consensus/src/threshold_simplex/actors/voter/actor.rs

```diff
@@ -368,39 +357,22 @@ impl<
                 "broadcasting notarization"
             );
 
+            // Only select notarizes that are for this proposal
+            let notarizes = notarizes
+                .iter()
+                .filter_map(|x| self.notarizes[*x as usize].as_ref());
+
             // Recover threshold signature
-            let proposal_signature = self
```

Comment:
Opted to remove this spawn code for now as data indicates there are much more efficient optimizations: #719